### DISCUSSION TITLE
Is it possible to change thumbnails size on Mosaic view

### DISCUSSION THREAD

**Discussion by farnots** [photoprism/photoprism#2023] [Q&A] [answered]
Hello I found the Mosaic to have too little thumbnail for images. 

I'm wondering if it's something we can edit ? I don't use much the cards view even if it has bigger thumbnail because there is lot of information I don't need when browsing my library.

Thanks

**Comment by graciousgrey** [comment, marked as answer]
Hello,

Not yet. We do have an open issue for this: https://github.com/photoprism/photoprism/issues/645

**Reply by farnots** [reply]
Nice I didn't see it. Thanks
